src/@carbon-info/views/Ecosystem/components/AppsCarousel.tsx

```diff
@@ -50,7 +52,7 @@ const AppsCarousel: React.FC<Props> = (props: Props) => {
             </Box >
           </Grow>
         ) : (
-          <Grow in timeout={(index + 1) * 200 > 1000 ? 1000 : (index + 1) * 200} key={`${name}-featured-dApps`}>
+          <Grow in timeout={(index + 1) * 200 > 1000 ? 2000 : (index + 1) * 2000} key={`${name}-featured-dApps`}>
```

Comment:
Think this implementation is not correct: this is causing the cards to appear very sluggishly.